GameOfLife/displaygameboardonthecommandline.h

```diff
@@ -5,9 +5,12 @@
 
 class DisplayGameBoardOnTheCommandLine
 {
+private:
+    int coutnerOfIteration;
 public:
     DisplayGameBoardOnTheCommandLine();
     void displayGameBoardOnTheCommandLine(std::string stringConvertedFromBoard);
+    int getCoutnerOfIteration();
 };
```

Comment:
Counter not Coutner!
And plural Iterations, because there will be more than one